include/swift/AST/DiagnosticsSIL.def

```diff
@@ -179,7 +179,11 @@ ERROR(assignment_to_immutable_value,none,
 ERROR(missing_return,none,
       "missing return in a %select{function|closure}1 expected to return %0",
       (Type, unsigned))
-ERROR(non_exhaustive_switch,none, 
+ERROR(missing_never_call,none,
+      "never-returning %select{function|closure}0 is missing call to "
+      "another never-returning function on all paths.",
+      (unsigned))
```

Comment:
I think if I change `isNever` to `isUninhabited` and put back the original implementation I can make this

> function with uninhabited return type is missing call to never-returning function on all paths